openapi/Swarm.yaml

```diff
@@ -0,0 +1,214 @@
+openapi: 3.0.0
+info:
+  version: 0.1.0
+  title: Swarm API
+  description: 'A list of the currently provided Interfaces to interact with the swarm, implementing file operations'
+security:
+  - {}
+externalDocs:
+  description: Read the documentation @ Swarm-guide
+  url: 'https://swarm-guide.readthedocs.io/en/latest/'
+servers:
+  - url: '{apiRoot}/v1'
+    variables:
+      apiRoot:
+        default: 'http://localhost:8080'
+        description: Base address of the local swarm node API
+paths:
+  
+  '/bytes':
+    post:
+      summary: 'Upload data'
+      tags: 
+        - 'Endpoints on local swarm node'
+      requestBody:
+        content:
+          application/octet-stream:
+            schema:
+              type: string
+              format: binary
+      responses:
+        '200':
+          description: Ok
+          content:
+            application/json:
+              schema:
+                $ref: 'SwarmCommon.yaml#/components/schemas/Reference'
+        '500':
+          $ref: 'SwarmCommon.yaml#/components/responses/500'
+        default:
+          description: Default response
+
+  '/bytes/{address}':
+    get:
+      summary: 'Get addressed data'
```

Comment:
All data in Swarm is addressed. I would say: `get data, possibly consisting multiple chunks by it's root address`. Or something similar